.github/workflows/contracts.yml

```diff
@@ -11,6 +13,9 @@ on:
       - ".github/workflows/contracts.yml"
   pull_request:
     branches:
+      # TODO: Run on all branches or only on master (to be decided) 
+      # after we're fully migrated from Circle CI
+      - "rfc-18/**"
       - master
     paths:
       - "solidity/contracts/**"
```

Comment:
We should also add `solidity/contracts/test/**`. But maybe it's not worth listing all the specific paths (as in the future we will add `solidity/migrations/**`) and we could generalize it to `solidity/**`. If so we would have to exclude `solidity/dashboard`. So we would have:
```yml
- "solidity/**"
- "!solidity/dashboard/**"
- ".github/workflows/contracts.yml"
```